pkg/issuer/vault/issue.go

```diff
@@ -150,8 +225,7 @@ func (v *Vault) initVaultClient() (*vault.Client, error) {
 			return nil, fmt.Errorf("error reading Vault token from secret %s/%s: %s", v.resourceNamespace, tokenRef.Name, err.Error())
 		}
 		client.SetToken(token)
-
-		return client, nil
```

Comment:
Why don't we return here anymore?

Reply:
I didn't want to have to duplicate some of the code in both the tokenRef and approle if blocks; therefore, the return line is moved out of from those two blocks. it's on line 243 instead.